### PR TITLE
Hamzahsn/feat/show notification when not secure

### PR DESCRIPTION
**Pull request by hamzahsn**
This PR...

## Changes

- This PR will show a nice notification when the host protocol is not secure.

## Fixes

-

## How to test it

-

## screenshots

- 
Screenshot 2021-10-06 at 14 29 59

## Checklist

- [x ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by hamzahsn** [comment]
Need to confirm the styles and the text and then we can run it after the Get tests modal.

**Comment by olensmar** [comment]
it's not enough to show a notification - we need to tell users what to do about it... we'll need to create a page in the documentation that we can link to from here...

**Comment by hamzahsn** [comment]
So you suggest something like: `Dashboard is using non-secure protocol! find more here` ? 🤔

**Comment by hamzahsn** [comment]
Needs to update the link to the docs in the error notification as write now it's in progress.

File: src/App.tsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ function App() {
 
     if (!isHostProtocolSecure()) {
       showSmallError(`Dashboard is using non-secure protocol! Read more
```
Comment (olensmar) [inline src/App.tsx:112]: remove "Read more" on this line - not necessary to have it both with and without link

File: src/App.tsx [hunk 2]
```diff
@@ -110,7 +110,7 @@ function App() {
...
-      <a href='https://github.com/kubeshop/kubtest/blob/main/docs/getting-started.md' target="_blank" rel="noopener">Read more</a>`);
+      <a href='https://kubeshop.github.io/kubtest/installing/' target="_blank" rel="noopener">Read more</a>`);
```
Comment (olensmar) [inline src/App.tsx:113]: we will  update the link before the release - this is ok for now

File: src/App.tsx (hunk 1 above)
Comment (hamzahsn) [reply src/App.tsx:112]: ooops missed that

File: src/App.tsx (hunk 2 above)
Comment (olensmar) [reply src/App.tsx:113]: yes - we'll update when we have the content available

**Review by olensmar — APPROVED** [review]

**Pull request merged**